src/main/scala/viper/gobra/frontend/info/implementation/resolution/NameResolution.scala

```diff
@@ -284,8 +289,31 @@ trait NameResolution { this: TypeInfoImpl =>
 
       case n =>
         (n, lookup(sequentialDefenv(n), serialize(n), UnknownEntity())) match {
-          // in case no entity was found in the current package, look for it in unqualifiedly imported packages:
-          case (n: PIdnUse, UnknownEntity()) => tryUnqualifiedPackageLookup(n)
+
+          // lookup has failed
+          case (n: PIdnUse, UnknownEntity()) =>
+            // in case no entity was found in the current package, look for it in
+            // - unqualifiedly imported packages or
+            // - the current interface if `n` occurs in an interface definition
+            (tryUnqualifiedPackageLookup(n), tryEnclosingInterface(n)) match {
```

Comment:
I think this is a bit ugly because it does not scale if we increase the number of scopes, i.e. we do not want `(tryUnqualifiedPackageLookup(n), tryEnclosingInterface(n), tryX1(n), tryX2(n), tryX3(n), tryX4(n), ...)`